### DISCUSSION TITLE
有没有通过高德OD路径对OSM路网优化完善的方法

### DISCUSSION THREAD

**Discussion by Carrot0617** [zdsjjtTLG/TrackIt#7] [Q&A] [answered]
有没有通过高德OD路径对OSM路网优化完善的方法？我看到方法好像只有分别对OSM地图进行优化和利用高德OD构建路网的方法。但是我的路网出现了OSM路网可能不完备，缺少一些高德路径的部分。能否通过高德进行OSM路网完善？

**Comment by zdsjjtTLG** [comment, marked as answer]
不推荐将高德路径和OSM路网混用，他俩的建模还是有差异的，你可以试试这个思路：基于OSM路网去构造带途径点的OD，再去请求高德路径获取路网，当然除了自定OD外你也可以再用一些随机OD

**Reply by Carrot0617** [reply]
好的，谢谢
